tsconfig.json

```diff
@@ -18,7 +18,8 @@
     "paths": {
       "@lib/*": ["lib/*"],
       "@modules/*": ["modules/*"],
-      "@pages/*": ["pages/*"]
+      "@pages/*": ["pages/*"],
+      "@locales/*": ["../locales/*"]
```

Comment:
Definitely a matter of taste, but I usually recommend moving messages out of the src folder e.g. into `./messages` as they're not really source code.